paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/link/LinkConfirmationDefinition.kt

```diff
@@ -57,7 +58,8 @@ internal class LinkConfirmationDefinition @Inject constructor(
         launcher.present(
             configuration = confirmationOption.configuration,
             linkAccount = linkAccountHolder.linkAccount.value,
-            useLinkExpress = confirmationOption.useLinkExpress
+            useLinkExpress = confirmationOption.useLinkExpress,
+            launchMode = LinkLaunchMode.Payment
```

Comment:
feels like these two could be combined into a launchMode sealed class (where on PaymentMode `useLinkExpress` is conditional, but on AuthenticationMode we likely always want to present the 2FA eagerly so this should not be configurable)